dtm0/tx_desc.c

```diff
@@ -27,13 +27,33 @@
  * @{
  */
 
+#include "dtm0/clk_src.h"
 #define M0_TRACE_SUBSYSTEM M0_TRACE_SUBSYS_DTM
 #include "dtm0/tx_desc.h"
 #include "lib/assert.h" /* M0_PRE */
 #include "lib/memory.h" /* M0_ALLOC */
 #include "lib/errno.h"  /* ENOMEM */
 #include "lib/trace.h"  /* M0_ERR */
 
+M0_INTERNAL void m0_dtm0_tx_desc_init_none(struct m0_dtm0_tx_desc *td)
+{
+	td->dtd_id.dti_fid = M0_FID0;
+	td->dtd_id.dti_ts = M0_DTM0_TS_INIT;
+	td->dtd_pg.dtpg_nr = 0;
+	td->dtd_pg.dtpg_pa = NULL;
+}
+
+M0_INTERNAL bool m0_dtm0_tx_desc_is_none(const struct m0_dtm0_tx_desc *td)
```

Comment:
Let's use the simplest possible model: zeroed tx_desc == empty tx_desc:

```c
void init_none(td) {
  M0_SET0(td);
}
bool is_none(td) {
  return M0_IS0(td);
}
```

Reply:
Done.